app/controllers/admin/admin_base_controller.rb

```diff
@@ -25,5 +25,10 @@ def set_cache_buster
       response.headers['Pragma'] = 'no-cache'
       response.headers['Expires'] = 'Fri, 01 Jan 1990 00:00:00 GMT'
     end
+
+    def convert_date_params(params)
```

Comment:
This code exists in this class and `ProviderBaseController`. Would it be worth moving it to the `ApplicationController` so that we're only defining it once?

Reply:
👍